rust/datafusion/tests/sql.rs

```diff
@@ -2607,3 +2607,24 @@ async fn invalid_qualified_table_references() -> Result<()> {
     }
     Ok(())
 }
+
+#[tokio::test]
+#[cfg(feature = "regex_expressions")]
+async fn query_regexp_match() -> Result<()> {
```

Comment:
It think we can remove this and just add these cases to the `test_regex_expressions` function above:

```rust
    test_expression!("regexp_match('foobarbequebaz', '')", "[]");
    test_expression!("regexp_match('foobarbequebaz', '(bar)(beque)')", "[bar, beque]");
    test_expression!("regexp_match('foobarbequebaz', '(ba3r)(bequ34e)')", "NULL");
    test_expression!("regexp_match('aaa-0', '.*-(\\d)')", "[0]");
    test_expression!("regexp_match('bb-1', '.*-(\\d)')", "[1]");
    test_expression!("regexp_match('aa', '.*-(\\d)')", "NULL");
    test_expression!("regexp_match(NULL, '.*-(\\d)')", "NULL");
    test_expression!("regexp_match('aaa-0', NULL)", "NULL");
```

Reply:
done